src/Cli/Microsoft.TemplateEngine.Cli/TemplatePackageDisplayer.cs

```diff
@@ -0,0 +1,422 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Abstractions.Constraints;
+using Microsoft.TemplateEngine.Abstractions.Installer;
+using Microsoft.TemplateEngine.Abstractions.TemplatePackage;
+using Microsoft.TemplateEngine.Cli.Commands;
+using Microsoft.TemplateEngine.Cli.TabularOutput;
+using Microsoft.TemplateEngine.Edge;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Cli
+{
+    /// <summary>
+    /// The class is responsible for displaying template package manipulation results to console.
+    /// </summary>
+    internal class TemplatePackageDisplayer
```

Comment:
```suggestion
    internal class TemplatePackageDisplay
```

We already have similar class naming